mod/ckeditor/views/default/ckeditor/init.php

```diff
@@ -4,17 +4,20 @@
  * 
  * Doing this inline enables the editor to initialize textareas loaded through ajax
  */
-
+// @todo: use absolute path
+if (_elgg_view_may_be_altered('elgg/ckeditor/config.js', 'elgg/ckeditor/config.js')) {
+	elgg_deprecated_notice('elgg/ckeditor/config.js view has been deprecated. You should not overwrite it. Use "config, ckeditor" JS hook instead', '2.2');
+	?>
+	<script>
+		require(['elgg', 'elgg/ckeditor/config'], function(elgg, config) {
```

Comment:
This doesn't seem to guarantee `elgg/ckeditor/config` will load fast enough to be used in `elgg/ckeditor`.